ompi/mca/coll/libnbc/libdict/dict.c

```diff
@@ -16,24 +16,17 @@
 dict_malloc_func _dict_malloc = malloc;
 dict_free_func _dict_free = free;
 
-static void dict_destroy __P((dict *dct, int del));
-static void dict_itor_destroy __P((dict_itor *itor));
-static int  dict_int_cmp __P((const void *k1, const void *k2));
-static int  dict_uint_cmp __P((const void *k1, const void *k2));
-static int  dict_long_cmp __P((const void *k1, const void *k2));
-static int  dict_ulong_cmp __P((const void *k1, const void *k2));
-static int  dict_str_cmp __P((const void *k1, const void *k2));
-
-static dict_malloc_func
-dict_set_malloc(dict_malloc_func func)
+
+dict_malloc_func
+ompi_mca_coll_libnbc_dict_set_malloc(dict_malloc_func func)
 {
 	dict_malloc_func old = _dict_malloc;
 	_dict_malloc = func ? func : malloc;
 	return old;
 }
 
-static dict_free_func
-dict_set_free(dict_free_func func)
+dict_free_func
```

Comment:
Are these functions used outside of this file? If not we shouldn't remove the `static` qualifier.

Reply:
They are not used anywhere so changed back to static. Since they are static and not used anywhere this will result in compiler warnings about unused functions.